dags/taar_weekly.py

```diff
@@ -6,55 +6,77 @@
 from datetime import datetime, timedelta
 from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
 from utils.mozetl import mozetl_envvar
+from airflow.operators.subdag_operator import SubDagOperator
```

Comment:
nit: imports arent grouped together or in any alphabetical ordering

Reply:
Thx.  Fixed in 49c1c930